### PR TITLE
Add optical_flow definitions & add Dshot support into MATEKF411

### PR DESCRIPTION
**Pull request by MATEKSYS**

### CODE REVIEW

File: src/main/target/MATEKF405/target.h [hunk 1]
```diff
@@ -176,6 +176,9 @@
 
 #define PITOT_I2C_BUS               DEFAULT_I2C_BUS
 
+#define USE_OPTICAL_FLOW
```
Comment (shellixyz) [inline src/main/target/MATEKF405/target.h:179]: `USE_OPTICAL_FLOW` is not a used macro, maybe you meant `USE_OPFLOW` ? In this case it is already defined for all targets having more than 256k flash, it is not needed.

File: src/main/target/MATEKF405/target.h [hunk 2]
```diff
@@ -176,6 +176,9 @@
...
+#define USE_OPFLOW_MSP
```
Comment (shellixyz) [inline src/main/target/MATEKF405/target.h:180]: `USE_OPFLOW_MSP` is already defined for all targets having more than 256k flash, it is not needed.

File: src/main/target/MATEKF405SE/target.h [hunk 3]
```diff
@@ -75,13 +75,16 @@
...
 #define USE_RANGEFINDER
+#define USE_RANGEFINDER_MSP
 #define USE_RANGEFINDER_HCSR04_I2C
 #define RANGEFINDER_I2C_BUS     BUS_I2C2
 
 #define PITOT_I2C_BUS           BUS_I2C2
 
 #define TEMPERATURE_I2C_BUS     BUS_I2C2
 
+#define USE_OPTICAL_FLOW
```
Comment (shellixyz) [inline src/main/target/MATEKF405SE/target.h:86]: `USE_OPTICAL_FLOW` is not a used macro, maybe you meant `USE_OPFLOW` ? In this case it is already defined for all targets having more than 256k flash, it is not needed.

File: src/main/target/MATEKF405SE/target.h [hunk 4]
```diff
@@ -75,13 +75,16 @@
...
+#define USE_OPFLOW_MSP
```
Comment (shellixyz) [inline src/main/target/MATEKF405SE/target.h:87]: `USE_OPFLOW_MSP` is already defined for all targets having more than 256k flash, it is not needed.

File: src/main/target/MATEKF722/target.h [hunk 5]
```diff
@@ -75,6 +75,14 @@
 
 #define PITOT_I2C_BUS           BUS_I2C1
 
+#define USE_RANGEFINDER
```
Comment (shellixyz) [inline src/main/target/MATEKF722/target.h:78]: `USE_RANGEFINDER`, `USE_RANGEFINDER_HCSR04_I2C`, `USE_OPFLOW` and `USE_OPFLOW_MSP` are already defined for all targets having more than 256k flash, it is not needed.

**Comment by MATEKSYS** [comment]
@shellixyz Gotcha, Pls close it. Thanks
 could you add the USD_DSHOT into F411 target

**Comment by leeph** [comment]
Thank you for this guys.

**Pull request closed without merging**